libs.autobuild

```diff
@@ -20,6 +20,15 @@ cmake_package "drivers/glfw" do |pkg|
 end
 remove_from_default 'drivers/glfw'
 
+import_package 'drivers/imu_myahrs_plus' do |pkg|
+    pkg.post_import do
+        Autobuild::Subprocess.run(pkg, 'build', 'make', :working_directory => pkg.srcdir + "/common_cpp")
```

Comment:
There is no makefile, cmake, or  file to build from the root directory. I need to go inside the tree folder to run the build process.